### PR TITLE
Add stupid quiet command to silence overly noisy tools

### PR DESCRIPTION
**Pull request by mat007**
Might not be excessively useful but at least the implementation is straightforward…
Does this need at test?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +67,17 @@ Example:
...
 
+### quiet
+```
+stupid quiet
+```
+Discards everything received on the standard input.
+
+Example:
+```
+stupid rm build | stupid quiet
```
Comment (jeanlaurent) [inline README.md]: `stupid rm build | stupid silence` is more funny I guess.

**Comment by jeanlaurent** [comment]
I guess `stupid silence` sounds better than `stupid quiet`

**Comment by vdemeester** [comment]
> Am I wrong to ask to change quiet by silence ?

You are not wrong to ask :stuck_out_tongue_closed_eyes:

**Comment by mat007** [comment]
PTAL!

**Pull request merged**